policyengine_us/reforms/ctc/eppc/expanded_ctc.py

```diff
@@ -0,0 +1,423 @@
+        label = "non-refundable tax credits"
+        unit = USD
+        documentation = "Capped value of non-refundable tax credits"
+        definition_period = YEAR
+        adds = ["income_tax_non_refundable_credits_pre_ctc"]
+        subtracts = ["income_tax_unavailable_non_refundable_credits"]
+
+    class income_tax_unavailable_non_refundable_credits(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "unavailable non-refundable tax credits"
+        unit = USD
+        documentation = "Total value of non-refundable tax credits that were not available to the filer due to having too low income tax."
+        definition_period = YEAR
+
+        def formula(tax_unit, period, parameters):
+            return -min_(
+                tax_unit("income_tax_before_credits", period),
+                tax_unit("income_tax_non_refundable_credits_pre_ctc", period),
+            ) + tax_unit("income_tax_non_refundable_credits_pre_ctc", period)
+
+    class income_tax_pre_ctc(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        unit = USD
+        label = "Federal income tax"
+        documentation = "Total federal individual income tax liability."
+
+        def formula(person, period, parameters):
+            if parameters(
+                period
+            ).gov.contrib.ubi_center.flat_tax.abolish_federal_income_tax:
+                return 0
+            else:
+                added_components = add(
+                    person, period, ["income_tax_before_refundable_credits"]
+                )
+                subtracted_components = add(
+                    person, period, ["income_tax_refundable_credits_pre_ctc"]
+                )
+                return added_components - subtracted_components
+
+    class income_tax(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        unit = USD
+        label = "Federal income tax"
+        documentation = "Total federal individual income tax liability."
+
+        def formula(tax_unit, period, parameters):
+            pre_ctc_tax = tax_unit("income_tax_pre_ctc", period)
+            indiv_tax = tax_unit("income_tax_pre_ctc_indiv", period)
+            filing_status = tax_unit("filing_status", period)
+            is_joint = filing_status == filing_status.possible_values.JOINT
+            non_ref_ctc = tax_unit("non_refundable_ctc", period)
+            ref_ctc = tax_unit("refundable_ctc", period)
+            regular_tax = max_(pre_ctc_tax - non_ref_ctc, 0) - ref_ctc
+            total_indiv_tax = max_(indiv_tax - non_ref_ctc, 0) - ref_ctc
+            smaller_tax = min_(total_indiv_tax, regular_tax)
+            return where(is_joint, smaller_tax, regular_tax)
+
+    class maximum_benefits(Variable):
+        value_type = float
+        entity = TaxUnit
+        definition_period = YEAR
+        label = "Maximum benfit amount which the housheold is entitled to"
```

Comment:
But again this requires a clear list of benefits. Please ask Jamie to write a more specific ruleset to add to the README.md
```suggestion
        label = "Maximum benefit amount to which the household is entitled"
```